src/commands/heroku.rs

```diff
@@ -0,0 +1,66 @@
+use heroku_rs::client::{Executor, Heroku};
+
+use serde_json::Value;
+
+use serenity::framework::standard::{macros::command, Args, CommandResult};
+use serenity::model::prelude::*;
+use serenity::prelude::*;
+
+use crate::config::Config;
+
+#[derive(Debug)]
+struct HerokuApp {
+    id: String,
+    name: String,
+}
+
+#[command]
+pub fn get_apps(ctx: &mut Context, msg: &Message, _args: Args) -> CommandResult {
+    let response = heroku_client().get().apps().execute::<Value>();
```

Comment:
Is there a reason why you're getting a `Value` instead of a `Vec<HerokuApp>`? If you add `#[derive(Deserialize)]` to `HerokuApp` you can just use:

```suggestion
    let response = heroku_client().get().apps().execute::<Vec<HerokuApp>>();
```

...and remove all the custom deserialization code.

Reply:
Sounds great - adding that in!

Reply:
Done!